scenarios/har/1/openapi.yaml

```diff
@@ -201,7 +201,6 @@ paths:
           required: true
           schema:
             type: integer
-            format: int64
```

Comment:
See #19 

For now I'm just going to say that we don't support custom formats. 

We will add them once we feel that ajv works for our use case.